docs/src/Tooltip/index.js

```diff
@@ -0,0 +1,120 @@
+          </p>
+          <p>
+            The tooltip allows users to set the position and anchor. Position refers to the position of the tooltip relative to its trigger (which is its children), and acceptable values are top, right, bottom, and left. Anchor refers to where the tooltip is anchored on the triggered element, and acceptable values are start, center, and end. Think of flexbox alignment when thinking about the start and end values.
+          </p>
+          <p>
+            View component source <a href="https://github.com/mesosphere/reactjs-components/blob/master/src/Tooltip/Tooltip.js">here</a>.
+            View full example source <a href="https://github.com/mesosphere/reactjs-components/blob/master/docs/src/Tooltip/index.js">here</a>.
+          </p>
+          <h3>Properties API</h3>
+          <div className="example-block">
+            <pre className="prettyprint linenums flush-bottom">
+{`Tooltip.propTypes = {
+  // Anchor the tooltip to an edge of the tooltip trigger. Start and end refer
+  // to the logical respective edges of the tooltip. When tooltip is positioned
+  // on the right, start refers to the top of the tooltip. When tooltip is
+  // positioned on the bottom, start refers to the left edge of the tooltip.
+  anchor: React.PropTypes.oneOf(['start', 'center', 'end']),
+  // The children will be used as the trigger.
+  children: React.PropTypes.node.isRequired,
+  className: React.PropTypes.string,
+  // The tooltip's content.
+  content: React.PropTypes.node.isRequired,
+  // Position the tooltip on an edge of the tooltip trigger. Default is top.
+  position: React.PropTypes.oneOf(['top', 'bottom', 'right', 'left']),
+  tooltipWrapperClassName: React.PropTypes.string,
+  // Explicitly set the width of the tooltip. Default is auto.
+  width: React.PropTypes.number,
+  // Allow the text content to wrap. Default is false. This should be used with
+  // the width property, because otherwise the width of the content will be the
+  // same as the trigger.
+  wrapText: React.PropTypes.bool
+};`}
+            </pre>
+          </div>
+          <div className="example-block flush-bottom example-block-overflow">
+            <div className="example-block-content">
+              <section className="row canvas-pod">
+                <div className="column-12">
+                  <p>
+                    Hover over the following buttons to observe the behavior
+                    of the tooltips.
+                  </p>
+                </div>
+              </section>
+              <section className="row canvas-pod">
+                <div className="column-12 column-mini-12">
+                  <Tooltip content="I'm a tooltip!">
+                    <button className="button">Top</button>
+                  </Tooltip>
+                  <Tooltip content="I'm a tooltip!" position="right">
+                    <button className="button">Right</button>
```

Comment:
This is kind of silly but maybe we could switch the right and left tooltips, because when hovering over buttons left-to-right or right-to-left the tooltips get in the way of the next button and you have to mouseoff one button to get to the next button.